server.go

```diff
@@ -1177,6 +1181,35 @@ func startWebgo() {
 	router.Start()
 }
 
+// Goyave
+func goyaveHandler(r *goyave.Response, req *goyave.Request) {
+	if cpuBound {
+		pow(target)
+	} else {
+
+		if sleepTime > 0 {
+			time.Sleep(sleepTimeDuration)
+		} else {
+			runtime.Gosched()
+		}
+	}
+	r.String(http.StatusOK, "hello")
+}
+
+func getGoyaveRoutes(router *goyave.Router) {
+	router.Get("/hello", goyaveHandler)
+}
+
+func startGoyave() {
+	if err := config.LoadFrom("goyavecfg.json"); err != nil {
```

Comment:
Should it parse a template with args instead of a config file?

Reply:
This framework is opinionated and required a configuration file. (By default it uses `config.json` in the working directory, but you can manually load one too)

Reply:
Alternatively to avoid problems with missing files, I can create a temp file and load it.